### PR TITLE
Fix flakiness test

### PR DESCRIPTION
**Pull request by jadewang-db**
fix the flakiness server test by only random query status for a specific table which which used by the rest client.

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.0"
+version in ThisBuild := "1.1.1-SNAPSHOT"
```
Comment (linzhou-db) [inline version.sbt:1]: revert this?

File: server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala [hunk 2]
```diff
@@ -364,9 +364,10 @@ class DeltaSharingService(serverConfig: ServerConfig) {
     }
 
     // simulate async query with 50% chance of return
-    // asynchronously client should be able to hand both cases
-    // this is for test purpose only and shouldn't cause flakiness
-    if(rand.nextInt(100) > 50) {
+    // asynchronously for a specific table
+    // client should be able to hand both cases and for server
```
Comment (linzhou-db) [inline server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala:368]: nit: hand -> handle.

File: client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala [hunk 3]
```diff
@@ -166,7 +166,7 @@ class DeltaSharingRestClient(
     asyncQueryMaxDuration: Long = 600000L
   ) extends DeltaSharingClient with Logging {
 
-  logError(s"DeltaSharingRestClient with enableAsyncQuery $enableAsyncQuery")
+  logInfo(s"DeltaSharingRestClient with enableAsyncQuery $enableAsyncQuery")
```
Comment (linzhou-db) [inline client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala:169]: is this intended?

Comment (jadewang-db) [reply client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala:169]: yes

**Review by linzhou-db — APPROVED** [review]
Thanks for the fix!

**Pull request merged**